concourse/pipelines/deploy.yml

```diff
@@ -150,6 +150,23 @@ resources:
         - terraform/modules/task-definition
         - terraform/modules/task-definitions/signon
 
+  - <<: *git-repo
+    name: smokey
+    source:
+      branch: main
+      uri: https://github.com/alphagov/smokey
+      tag_filter: "*-release"
```

Comment:
I'm not completely happy with the tagging, we could make a change to the semver resource to support version prefixes (e.g. "v" or "release_"). At the moment adding a suffix to a tag (i.e. "1.2.3-release") is the best the resource can do.

Reply:
It would probably be nicer to use a `registry-image` resource for smokey, instead of cloning the git repo just to work out what the tag of the image in the registry is 😄. I understand that it has to be like this for now to be compatible with the other apps though, so I think this is fine.